src/daily_output.ts

```diff
@@ -246,12 +227,13 @@ export class DailyOutput {
 		const node = edge.node;
 		const txId = node.id;
 		const ownerAddress = node.owner.address;
-		let tip = +node.quantity.winston;
+		const tip = +node.quantity.winston;
 		const tags = node.tags;
 		const dataSize = +node.data.size;
 		const entityTypeTag = tags.find((tag) => tag.name === 'Entity-Type')?.value;
 		const bundledIn = node.bundledIn?.id;
 		const bundleVersion = tags.find((tag) => tag.name === 'Bundle-Version')?.value;
+		const bundleTipType = tags.find((tag) => tag.name === 'Tip-Type')?.value;
 		const isMetadataTransaction = !!entityTypeTag && !bundleVersion;
 		const isBundleTransaction = !!bundleVersion;
```

Comment:
We could improve the locality for some these variables by moving them to where they are used. It is difficult to mentally track all of these constants up front

Reply:
I do add more variables at #15. I'm gonna do the cleanup there